src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -204,53 +235,154 @@ private void Uninstall()
                     dynamoViewModel.BrandingResourceProvider.ProductName),
                     Resources.UninstallFailureMessageBoxTitle,
                     MessageBoxButton.OK, MessageBoxImage.Error);
             }
             finally
             {
-                RaisePropertyChanged(nameof(PackageLoadStateTooltip));
-                RaisePropertyChanged(nameof(PackageLoadStateText));
+                NotifyLoadStatePropertyChanged();
             }
         }
 
         private bool CanUninstall()
         {
-            if (!Model.InUse(dynamoViewModel.Model) || Model.LoadedAssemblies.Any())
+            if (!Model.InUse(dynamoModel) || Model.LoadedAssemblies.Any())
             {
-                return Model.BuiltInPackage ? 
+                return Model.BuiltInPackage ?
                     Model.LoadState.State != PackageLoadState.StateTypes.Unloaded &&
                     Model.LoadState.ScheduledState != PackageLoadState.ScheduledTypes.ScheduledForUnload :
                     Model.LoadState.ScheduledState != PackageLoadState.ScheduledTypes.ScheduledForDeletion;
             }
             return false;
         }
 
-        private void GoToRootDirectory()
+        // Loads a built-in package that was previously set as Unloaded
+        private void Load()
+        {
+            var packageLoader = dynamoModel.GetPackageManagerExtension().PackageLoader;
+            var conflicts = packageLoader.LocalPackages.ToList().Where(x => x.Name == Model.Name && x != Model);
+            bool hasConflictsWithLoadedAssemblies = conflicts.Any(x => x.InUse(dynamoModel));
+
+            if (conflicts.Any())
+            {
+                string conflictsMsg = string.Join(",", conflicts.Select(x => x.Name + " " + x.VersionName));
+
+                string message = hasConflictsWithLoadedAssemblies ? 
+                        string.Format(Resources.MessageLoadBuiltInPackageWithRestart,
+                        conflictsMsg, Model.Name + " " + Model.VersionName) 
+                        :
+                        string.Format(Resources.MessageLoadBuiltInPackage,
+                        conflictsMsg, Model.Name + " " + Model.VersionName);
+
+                var dialogResult = MessageBoxService.Show(message,
+                    Resources.CannotDownloadPackageMessageBoxTitle,
+                        MessageBoxButton.OKCancel,
+                        MessageBoxImage.Exclamation);
+
+                if (dialogResult == MessageBoxResult.Cancel) return;
+            }
+
+            try
+            {
+                foreach (var package in conflicts)
+                {
+                    package.UninstallCore(dynamoModel.CustomNodeManager, packageLoader, dynamoModel.PreferenceSettings);
+                }
+
+                if (!hasConflictsWithLoadedAssemblies)
+                {
+                    try
+                    {
+                        packageLoader.TryLoadPackageIntoLibrary(Model);
```

Comment:
@pinzart90 New problem. Here is what I did with a single singed package installed at the built-in location:
1. Started Dynamo and marked the package for `Unload`.
2. Restarted Dynamo
3. The package is now labeled as `Unloaded`. I mark it for `Load`. 
The package is now labeled as `Loaded` but is not actually loaded (It does not show up in the library). The call to `TryLoadPackageIntoLibrary` does nothing.
And finally, the package end up as `Unloaded` again if I restart `Dynamo` as this point.

Reply:
Nice catch. Should be fixed now.
Will try to add test coverage for this

Reply:
Works fine now!